packages/runtime/src/components/builtin/Slot/register.ts

```diff
@@ -0,0 +1,20 @@
+import { lazy } from 'react';
+import { ReactRuntime } from '../../../react'
+import { MakeswiftComponentType } from '../constants'
+import { Checkbox } from '@makeswift/controls'
+import { Slot } from '../../../controls';
+
+export function registerComponent(runtime: ReactRuntime) {
+  return runtime.registerComponent(
+    lazy(() => import('./SlotRoot')),
+      {
+        type: MakeswiftComponentType.Slot, 
+        label: 'Slot',
+        hidden: true,
+        props: {
+          children: Slot(),
+          showFallback: Checkbox({ label: "Name of checkbox test", defaultValue: true })
```

Comment:
I think this label needs to be updated

Reply:
Yikes. Thank you. Changed to 'Use Fallback' for now, but let me know if you prefer something else.

Reply:
Yikes. Changed to 'Use Fallback' for now, but let me know if you'd prefer a different label.